tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -561,6 +564,73 @@ public function testAlterTableAutoIncrementIntToBigInt(
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+    public function testListTableColumnsOidConflictWithNonTableObject(): void
+    {
+        $semanticVersion = $this->connection->fetchOne('SHOW SERVER_VERSION');
```

Comment:
Doesn't this work?

```suggestion
        $semanticVersion = $this->connection->getServerVersion();
```

Reply:
`getServerVersion` on the connection object is private.  `$this->connection->getWrappedConnection()` has a public method, but is deprecated so I went with`$this->connection->getNativeConnection()->getAttribute(PDO::ATTR_SERVER_VERSION)`.

Reply:
> `getServerVersion` on the connection object is private.

Indeed.

> `$this->connection->getWrappedConnection()` has a public method, but is deprecated

Let's use that anyway. I'll have to adjust the call in 4.0.x, but that's fine.

> so I went with`$this->connection->getNativeConnection()->getAttribute(PDO::ATTR_SERVER_VERSION)`.

This will fail as soon as we merge your change to 3.6.x because we have a Postgres driver there that does not use PDO.